virtuoso.sh

```diff
@@ -1,4 +1,7 @@
 #!/bin/bash
+SETTINGS_DIR=/settings
```

Comment:
It's a nitpick: but why make this configurable?
This is essentially internal housekeeping, no need to expose that in a "public API".
If it's not configurable, you can also ensure that this directory exists in the Dockerfile, rather than the startup script.